doc/source/cluster/kubernetes/examples/rayjob-kueue-priority-scheduling.md

```diff
@@ -92,7 +55,6 @@ metadata:
   name: "cluster-queue"
 spec:
   preemption:
-    reclaimWithinCohort: Any
```

Comment:
We don't set Cohort for the ClusterQueue, so we don't need `reclaimWithinCohort`.